CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+v0.4.5-RC1 / 2019-12-05
+=======================
+  
+  * fix security vulnerability in images used in cleanup pods
```

Comment:
At the end of each item, can we add a list of PRs or Issues, followed by the contributor github handle.